app/viewModels/P5/departure/IncidentP5ViewModel.scala

```diff
@@ -66,17 +67,25 @@ object IncidentP5ViewModel {
       incidentIndex: Index
     )(implicit messages: Messages, ec: ExecutionContext, hc: HeaderCarrier): Future[IncidentP5ViewModel] = {
 
-      val helper = new IncidentP5Helper(data.Consignment.Incident(incidentIndex.position), referenceDataService)
+      val incident = data.Consignment.Incident(incidentIndex.position)
+      val helper   = new IncidentP5Helper(incident, referenceDataService)
+      val transhipmentHelper = incident.Transhipment.map(
+        transhipment => new IncidentP5TranshipmentHelper(transhipment, incidentIndex.display, referenceDataService)
+      )
+
+      val incidentInformationSectionFuture                                 = helper.incidentInformationSection
+      val transhipmentSectionFuture: Future[Option[Section.StaticSection]] = transhipmentHelper.map(_.replacementMeansOfTransportSection).sequence
```

Comment:
We can avoid needing to map twice by just pulling `transhipmentHelper` into here

```
      val transhipmentSectionFuture: Future[Option[StaticSection]] =
        incident.Transhipment.map {
          transhipment =>
            val helper = new IncidentP5TranshipmentHelper(transhipment, incidentIndex.display, referenceDataService)
            helper.replacementMeansOfTransportSection
        }.sequence
```